src/ui_fsmenu/main.cc

```diff
@@ -474,7 +474,7 @@ void MainMenu::set_labels() {
 
 	version_.set_text(
 	   /** TRANSLATORS: %1$s = version string, %2%s = "Debug" or "Release" */
-	   format(_("Version %1$s (%2$s)"), build_id(), build_type()));
+	   format(_("Version %1$s"), build_ver_details()));
```

Comment:
What should it say? I don't even understand why it's there. To me, it looks like the only string that can be translated there is "Version".

Reply:
The comment is to give translators a hint what is semantically placed in the placeholder. Some languages might want to change the order ("%1$s Version ") or something different, so that's why there is this formatting.
```
/** TRANSLATORS: %1$s = build version and type */
```

Reply:
The comment is to give translators a hint what is semantically placed in the placeholder. Some languages might want to change the order ("%1$s Version ") or something different, so that's why there is this formatting.
```
/** TRANSLATORS: %1$s = build version and type */